sdk/core/test/helpers/getOtpFromTestMailApp.ts

```diff
@@ -0,0 +1,46 @@
+let fetch: any
+
+/* istanbul ignore next */
+if (!fetch) {
+  fetch = require('node-fetch')
+}
+
+export class EmailConfiguration {
+  public static readonly TESTMAIL_URL = 'https://api.testmail.app'
+  public static readonly TESTMAIL_API_KEY = process.env.TESTMAIL_API_KEY
+  public static readonly TESTMAIL_NAMESPACE = process.env.TESTMAIL_NS
+  public static readonly TESTMAIL_PATH = '/api/json'
+}
+
+export interface TestEmail {
+  subject: string
+  html: string
+  text: string
+  downloadUrl: string
+  from: string
+  to: string
+  tag: string
+  timestamp: number
+}
+
+export class TestEmailHelper {
+  public static generateEmailForTag = (tag: string): string =>
+    `${EmailConfiguration.TESTMAIL_NAMESPACE}.${tag}@inbox.testmail.app`
+  public static getEmailsForTag = async (tag: string): Promise<TestEmail[]> => {
+    const url = `${EmailConfiguration.TESTMAIL_URL}${EmailConfiguration.TESTMAIL_PATH}?apikey=${EmailConfiguration.TESTMAIL_API_KEY}&namespace=${EmailConfiguration.TESTMAIL_NAMESPACE}&tag=${tag}`
```

Comment:
Maybe better to use `querystring.stringify` and `new URL` there, instead of building it all by hand (and not escaping the parameters properly)?

Also, what's the point of separating `TESTMAIL_URL` and `TESTMAIL_PATH` into two different config parameters? Are they going to ever be used separately?

Reply:
Let's keep stuff simple. I use Oleg's implementation as a reference. It is nice to have npm module in a future

Reply:
But with the proposed changes it would still be simple:

```
const urlObject = new URL(EmailConfiguration.TESTMAIL_URL)
urlObject.path = EmailConfiguration.TESTMAIL_PATH
urlObject.search = querystring.stringify({
  apiKey: EmailConfiguration.TESTMAIL_API_KEY,
  namespace: TESTMAIL_NAMESPACE,
  tag,
}
const url = urlObject.href
```

(Or without a second line, if we merge TESTMAIL_URL and TESTMAIL_PATH into a single string in config)